content/asset-tracking/using-the-sdks.textile

```diff
@@ -679,4 +679,5 @@ h2(#see-also). See also
 * "Overview":/asset-tracking
 * "Using the example apps":/asset-tracking/example-apps
 * "Asset Tracking Android training workshop project":https://github.com/ably/asset-tracking-android-training
-* "Reference documentation generated from SDK source code":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/index.html
+* "Ably Asset Tracking Android SDK Reference - generated from source code":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/index.html
+* "Ably Asset Tracking Swift SDKs Reference - generated from source code":https://sdk.ably.com/builds/ably/ably-asset-tracking-swift/main/jazzy/
```

Comment:
> Ably Asset Tracking Swift SDKs Reference - generated from source code

Rather than SDKs, I would make this SDK to match the Android version.

Reply:
Done